src/view/pages/NewCaseStudy/sections/Results.tsx

```diff
@@ -223,26 +238,40 @@ const StatHead = styled.div`
 
   @media ${MediaQuery.tablet} {
     width: 270px;
+    ${(props) => (props.statsType === StatsType.HORIZONTAL ? '' : 'display: flex')}
+  }
+  @media ${MediaQuery.desktopS} {
+    font-size: ${(props) => (props.statsType === StatsType.HORIZONTAL ? '90px' : '65px')};
+    ${(props) => (props.statsType === StatsType.HORIZONTAL ? '' : 'display: flex')}
   }
 `;
 
-const StatNumber = styled.p`
+const StatNumber = styled.p<{ color?: string; type?: number; statsType?: number;}>`
+  color: ${(props) => (props.statsType === StatsType.HORIZONTAL ? Palette.CYAN_BLUE : Palette.BLACK)};
   font-family: "Poppins-Medium";
-  font-size: 65px;
+  font-size: ${(props) => (props.statsType === StatsType.HORIZONTAL ? '80px' : '50px')};
   margin-left: 0px;
+  ${(props) => (props.statsType === StatsType.HORIZONTAL ? 'justify-content: space-around' : '' )};
 
   @media ${MediaQuery.tablet} {
-    margin-left: -25px;
+    font-size: ${(props) => (props.statsType === StatsType.HORIZONTAL ? '90px' : '65px')};
+    margin-left: ${(props) => (props.statsType === StatsType.HORIZONTAL ? '-25px' : '0px')};
   }
-`;
+  
+  @media ${MediaQuery.desktopS} {
+    font-size: ${(props) => (props.statsType === StatsType.HORIZONTAL ? '90px' : '65px')};
+  }
+  `;
 
-const StatTitle = styled.p`
+const StatTitle = styled.p<{ color?: string; statsType?: number; }>` 
   font-family: "Poppins-Medium";
   font-size: 20px;
   margin-bottom: 0px;
+  text-align: ${(props) => (props.statsType === StatsType.HORIZONTAL ? 'center' : 'inherit')};
+  ${(props) => (props.statsType === StatsType.HORIZONTAL ? 'margin-right: 25px' : '')};
```

Comment:
Remove `margin-right` entirely, replace `inherit` with `initial` since we want to reset text-align to its default value, and add `width: 100%;` so we align the text center. 

Also, add `text-align: center;` for tablet screens and larger since we can align the StatTitle center for the Poppy case study.

```suggestion
  text-align: ${(props) => (props.statsType === StatsType.HORIZONTAL ? 'center' : 'initial')};
  width: 100%;
  @media ${MediaQuery.tablet} {
    text-align: center;
  };
```